doc/examples/examples.hpp.in

```diff
@@ -196,14 +249,14 @@ OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  * <table align="center" width="90%">
  *
  *   <tr valign="top">
- *     <td> Solving a simple linear system with choice of executors.
+ *     <td> Solving a simple linear system with choice of executors
  *     </td>
  *     <td>@ref simple_solver
  *     </td>
  *   </tr>
  *
  *   <tr valign="top">
- *     <td> Debug the performance of a solver using loggers.
+ *     <td> Debug the performance of a solver using loggers
  *     </td>
  *     <td>@ref performance_debugging
  *     </td>
```

Comment:
Maybe we can add `preconditioner_export` here? It's not strictly performance debugging, but it's debugging for sure.
```suggestion
 *     <td> Debug the performance of a solver or preconditioner.
 *     </td>
 *     <td>@ref performance_debugging
 *         @ref preconditioner_export
 *     </td>
```